### PR TITLE
add new tests for proofs of origin and multiple duties

### PR DESCRIPTION
**Pull request by bhavanibandla**
### Jira link

- HOTT-2572
- HOTT-2352

### What?

I have added/removed/altered:

- Enabled **reintroducePsrRoO** spec file
- Added new tests for proofs of origin changes and multiple duties

### Why?

I am doing this because:

- To run new tests as part of the regression suite to ensure that all functionalities are working as expected

### CODE REVIEW

File: cypress/e2e/HOTT-Shared/importDuties.cy.js [hunk 1]
```diff
@@ -37,4 +37,26 @@ Move reassigned measure types (464, 481, 482, 483) from 'hide me' to import duti
     cy.contains('Import duties').click();
     cy.contains('Representative price').should('not.exist');
   });
+  it('UK third country duties - multiple duties - heading', function() {
+    cy.viewport(1200, 1080);
```
Comment (willfish) [inline cypress/e2e/HOTT-Shared/importDuties.cy.js:41]: Interesting. Are you okay to add a small comment explaining why this is necessary?

**Comment by willfish** [comment]
It looks good to me, I'm not sure if @jebw has managed to have a review, though?

File: cypress/e2e/HOTT-Shared/importDuties.cy.js (hunk 1 above)
Comment (bhavanibandla) [reply cypress/e2e/HOTT-Shared/importDuties.cy.js:41]: sure, I'll add a comment

Comment (bhavanibandla) [reply cypress/e2e/HOTT-Shared/importDuties.cy.js:42]: added comment

**Pull request merged**